build.sbt

```diff
@@ -2,21 +2,21 @@ name := "play-mailer"
 
 organization := "net.kaliber"
 
-scalaVersion := "2.11.6"
+scalaVersion := "2.11.8"
 
 resolvers ++= Seq(
   "Typesafe repository" at "http://repo.typesafe.com/typesafe/releases/",
   "scalaz-bintray" at "https://dl.bintray.com/scalaz/releases" /* specs2-core depends on scalaz-stream */
 )
 
-val playVersion = "2.4.0"
+val playVersion = "2.5.4"
```

Comment:
I think we should set this to `2.5.0` to make sure we accommodate all `2.5.x` users